src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1297,10 +1379,11 @@ public void AcceptNewConnection(RemoteConnection remoteConnection, EndPoint remo
                             break;
 
                         case InitialMessageType.AppendEntries:
-                            if (Follower.CheckIfValidLeader(this, remoteConnection, out var negotiation))
+                            var r = Follower.CheckIfValidLeader(this, remoteConnection);
```

Comment:
ideally this should be `async` as well (probably not in the scope of this issue)